planner/core/plan_to_pb.go

```diff
@@ -95,6 +95,9 @@ func (p *PhysicalTableScan) ToPB(ctx sessionctx.Context) (*tipb.Executor, error)
 		Columns: model.ColumnsToProto(p.Columns, p.Table.PKIsHandle),
 		Desc:    p.Desc,
 	}
+	if p.isPartition {
+		tsExec.TableId = p.physicalTableID
```

Comment:
I have a question, `TiFlash` doesn't use the `kv range`? since the `kv range` encoded with physical table ID.

Reply:
It's schemaful. TiFlash decode column with the table schema. So it needs a correct table id.